Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -4352,13 +4354,13 @@ void effectiveAppearanceDidChange(Inspector::ExtensionAppearance appearance) ove
 
         for (NSString *scriptPath : injectedContentData.scriptPaths) {
             RefPtr<API::Error> error;
-            auto *scriptString = extension->resourceStringForPath(scriptPath, error, WebExtension::CacheResult::Yes);
+            auto scriptString = extension->resourceStringForPath(scriptPath, error, WebExtension::CacheResult::Yes);
             if (!scriptString) {
                 recordError(::WebKit::wrapper(error));
                 continue;
             }
 
-            auto userScript = API::UserScript::create(WebCore::UserScript { scriptString, URL { m_baseURL, scriptPath }, makeVector<String>(includeMatchPatterns), makeVector<String>(excludeMatchPatterns), injectionTime, injectedFrames, waitForNotification }, executionWorld);
+            auto userScript = API::UserScript::create(WebCore::UserScript { WTFMove(scriptString), URL { m_baseURL, scriptPath }, makeVector<String>(includeMatchPatterns), makeVector<String>(excludeMatchPatterns), injectionTime, injectedFrames, waitForNotification }, executionWorld);
```

Comment:
`String` does not need a move, since it is a thin wrapper around `StringImpl`.

```suggestion
            auto userScript = API::UserScript::create(WebCore::UserScript { scriptString, URL { m_baseURL, scriptPath }, makeVector<String>(includeMatchPatterns), makeVector<String>(excludeMatchPatterns), injectionTime, injectedFrames, waitForNotification }, executionWorld);
```

Reply:
UserScript is a String&&, and without the move, the compiler complains about a missing rvalue. Generally, I've noticed other places that use a String&& move the string